### PR TITLE
Bump golang

### PR DESCRIPTION
**Pull request by dbadura**
**Description**

Changes proposed in this pull request:

- Bump golang to 1.18.1

**Related issue(s)**

### CODE REVIEW

File: resources/serverless/values.yaml [hunk 1]
```diff
@@ -95,7 +95,7 @@ global:
       version: "a56411a8"
     kaniko_executor:
       name: "kaniko-executor"
-      version: "1.8.0-ec157b48"
+      version: "PR-225"
```
[the remark this answers is not in the document]
Comment (dbadura) [reply resources/serverless/values.yaml:98]: I will use post-commit after merge of kaniko.

**Review by pPrecel — APPROVED** [review]

**Pull request merged**